Hast.Abstractions/Hast.Vitis.Abstractions/Docs/Zynq-PetaLinux.md

```diff
@@ -0,0 +1,164 @@
+ # Building PetaLinux 2020.2 SD card image for Hastlayer (Trenz TE0715-04-30-1C module)
+
+To be able to run Hastlayer accelerated .NET applications on Trenz Electronics TE0715-04-30-1C Zynq 7030 SOM SoC module you have to build PetaLinux with certain features enabled. This document describes the required steps.
+
+## Install Xilinx PetaLinux Tools 2020.2
+
+Download PetaLinux 2020.2 installer from Xilinx website (https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/embedded-design-tools/2020-2.html) and install it. 
+
+![Download PetaLinux Installer](Images/PetalinuxDownloadInstaller.png)
+
+PetaLinux building requires a 64-bit Linux machine with supported RedHat 7-8, CentOS 7-8, or Ubuntu 16-18-20. The actual PetaLinux tools installation must be done as a regular user, but you need root access to install additional packages. Note that you will need about 18GB temporary free disk space on the same drive to generate the output files. For more details see [the documentation](https://www.xilinx.com/content/dam/xilinx/support/documentation/sw_manuals/xilinx2021_1/ug1144-petalinux-tools-reference-guide.pdf).
+
+> ℹ️ If you are using the Centos 7 Docker image you don't have a regular root user. Perform the following to get yourself set up first:
+> ```shell
+> function user-setup() {
+>     # Creates non-root user and lets it take over /data.
+>     useradd user && passwd user && usermod -G wheel user || return
+>     chown user:user /data -R
+>      
+>     # Installs the necessary dependencies.
+>     yum install -y git diffstat unzip texinfo python chrpath wget xterm sdl rpcsvc-proto socat cpio inetutils python2 net-tools tftp-hpa python-virtualenv xorg-server-xvfb bison flex gnupg ncurses autoconf libtool tar gcc sdl sdl2 glib2 screen pax pax-utils libstdc++5 python-django iproute2 lib32-zlib openssl gawk python-pexpect python-pip python-gitpython python-jinja xz iputils python-pylint ncurses-devel gcc-c++
+>     locale-gen en_US.UTF-8 && update-locale
+>     echo 'export LANG=en_US.UTF-8' >> /home/user/.bashrc
+>     
+>     # Logs you in as "user".
+>     su user
+> }
+> 
+> user-setup
+> ```

```

Comment:
Ez olyen stílus amit szakmai tankönyvekben láttam gyakran (mondjuk jellemzően doboz szokott lenne de olyat standard MD nem tud) és itt pont hasznos. A konkrét formázást [ez a SO komment](https://stackoverflow.com/questions/25654845/how-can-i-create-a-text-box-for-a-note-in-markdown#comment110120466_43120795) javasolta.

Reply:
Aha, jól néz ki!